EventView-1/src/main/java/com/eventview/batch/ScheduleConfig.java

```diff
@@ -31,30 +22,15 @@
     @Autowired
     private UserService userService;
 
-    @Scheduled(cron = "0 0 1 * * *")
-    public void executes() throws ParseException {
-        List<Date> eventsList = eventService.getAllEventDates();
-        log.info("number of events{}", eventsList.size());
-        Calendar c = Calendar.getInstance();
-        Calendar d = Calendar.getInstance();
-        Calendar cal = Calendar.getInstance();
-        c.set(Calendar.DAY_OF_WEEK, Calendar.SUNDAY);
-        d.set(Calendar.DAY_OF_WEEK, Calendar.SATURDAY);
-        final DateFormat df = new SimpleDateFormat("yyyy-MM-dd");
-        Iterator iterator = eventsList.iterator();
-        int count = 1;
-        while (iterator.hasNext()) {
-            Date da = df.parse(iterator.next().toString());
-            cal.setTime(da);
-            log.info("list month{} month{}", cal.get(Calendar.MONTH), c.get(Calendar.MONTH));
-            if ((cal.get(Calendar.MONTH)) == (c.get(Calendar.MONTH))) {
-                if ((cal.get(Calendar.DAY_OF_MONTH)) == (c.get(Calendar.DAY_OF_MONTH))) {
-                    emailService.sendTextEmail(userService.getEmailByEvent(count),eventTypeService.getEventType(count),userService.getFNameByEvent(count),eventTypeService.getEventType(count));
-                    log.info("forwarded to email service c{} count{}", c.getTime(),count);
-                }
-            }
-            count++;
+    @Scheduled(cron = "0 0/08 20 * * *")
+    public void executes(){
+        Integer[] month = eventService.getAllEventMonths().toArray(new Integer[0]);
+        List<String> fNamesToday = userService.getFNameToday();
+        List<String> eventTypesToday = eventTypeService.getEventTypeToday();
+        List<String> fNamesWeek = userService.getFNameWeek();
+        List<String> eventTypesWeek = eventTypeService.getEventTypeWeek();
```

Comment:
why u need all these from line to 28-31

Reply:
I thought code would be easier to read this way